lib/server.js

```diff
@@ -24,13 +24,31 @@ function handleResponse (opts, req, resp, response) {
             response.headers = {};
         }
 
+        var rh = response.headers;
+
         // Set up CORS
-        response.headers['Access-Control-Allow-Origin'] = '*';
-        response.headers['Access-Control-Allow-Methods'] = 'GET';
-        response.headers['Access-Control-Allow-Headers'] = 'accept, content-type';
+        rh['Access-Control-Allow-Origin'] = '*';
+        rh['Access-Control-Allow-Methods'] = 'GET';
+        rh['Access-Control-Allow-Headers'] = 'accept, content-type';
+
+        // Set up security headers
+        // https://www.owasp.org/index.php/List_of_useful_HTTP_headers
+        rh['X-Content-Type-Options'] = 'nosniff';
+        rh['X-Frame-Options'] = 'SAMEORIGIN';
```

Comment:
Shouldn't this be set to `DENY`, and then if the response is not a JSON, set it to `SAMEORIGIN` in that case only?